actions/user_actions.jsx

```diff
@@ -33,27 +33,11 @@ export async function loadMe() {
 }
 
 export async function loadMeAndConfig(callback) {
-    const {data: config} = await getClientConfig()(store.dispatch, store.getState);
+    await getClientConfig()(store.dispatch, store.getState);
```

Comment:
I think we're slowly switching to the pattern of passing the action to `dispatch`, which should automatically pass the state, ala:
```
await store.dispatch(getClientConfig());
```